subsys/bluetooth/controller/ll_sw/ull.c

```diff
@@ -1793,11 +1799,13 @@ static inline int rx_demux_rx(memq_link_t *link, struct node_rx_hdr *rx)
 #if defined(CONFIG_BT_OBSERVER)
 #if defined(CONFIG_BT_CTLR_ADV_EXT)
 	case NODE_RX_TYPE_EXT_1M_REPORT:
-	case NODE_RX_TYPE_EXT_2M_REPORT:
```

Comment:
Why was only the 2M report removed here?

Reply:
yes, this is intentional, as 2M PHY is only used in auxiliary channels, lll_scan_aux does not generate `NODE_RX_TYPE_EXT_2M_REPORT`, but always generates `NODE_RX_TYPE_EXT_AUX_REPORT` (PHY value is read from the LLL context), and then the function `ull_scan_aux_setup` converts the node_rx to `NODE_RX_TYPE_EXT_2M_REPORT` if the auxiliary PDU was generated as part of Extended Scanning (and not a AUX_CHAIN_IND PDU of a Periodic Advertising Report).